src/pages/Gallery/Gallery.tsx

```diff
@@ -1,7 +1,118 @@
-import React from "react"
+import React, { useState } from "react"
+
+import styles from "./Gallery.module.scss"
+import classnames from "classnames"
+
+const Lines = () => {
+  return (
+    <div className={styles.lines}>
+      {Array.from({ length: 6 }).map((_, index) => (
+        <div key={index} className={styles.line} />
+      ))}
+    </div>
+  )
+}
 
 const Gallery = () => {
-  return <h1>Wojak gallery here</h1>
+  const [isFilterOpen, setIsFilterOpen] = useState(false)
+
+  return (
+    <>
+      <div>
+        <div className={styles.pageContainer}>
+          <div className={styles.galleryContainer}>
+            <div className={styles.headerContainer}>
+              <div>
+                <div className={styles.filterDropdownContainer}>
+                  <div>
+                    <button
```

Comment:
as I said, I would make the whole dropdown as standalone component with `onChange, placeholder, options` etc props
also it should be possible to close the dropdown by clicking outside of the dropdown :-)

Reply:
What about clicking on the dropdown to open it and close it as soon as the user moves the pointer outside of the component?